lnwallet/wallet.go

```diff
@@ -396,14 +396,31 @@ func (l *LightningWallet) Shutdown() error {
 	return nil
 }
 
-// ConfirmedBalance returns the current confirmed balance of the wallet. This
-// methods wraps the interal WalletController method so we're able to properly
-// hold the coin select mutex while we compute the balance.
+// ConfirmedBalance returns the current confirmed balance of the default
+// wallet account. This methods wraps the internal WalletController method so
+// we're able to properly hold the coin select mutex while we compute the
+// balance.
 func (l *LightningWallet) ConfirmedBalance(confs int32) (btcutil.Amount, error) {
 	l.coinSelectMtx.Lock()
 	defer l.coinSelectMtx.Unlock()
 
-	return l.WalletController.ConfirmedBalance(confs)
+	return l.WalletController.ConfirmedBalance(confs, DefaultAccountName)
+}
+
+// ListUnspentWitness returns all unspent outputs from the default wallet
+// account which are version 0 witness programs. The 'minConfs' and 'maxConfs'
+// parameters indicate the minimum and maximum number of confirmations an output
+// needs in order to be returned by this method. Passing -1 as 'minConfs'
+// indicates that even unconfirmed outputs should be returned. Using MaxInt32 as
+// 'maxConfs' implies returning all outputs with at least 'minConfs'.
+//
+// NOTE: This method requires the global coin selection lock to be held.
+func (l *LightningWallet) ListUnspentWitnessFromDefaultAccount(
```

Comment:
Is this preferable to just added the account as a default arg using a set of functional arguments? No strong pref on my end.

Reply:
That's what I had before. Having the name be explicit was suggested and preferred: https://github.com/lightningnetwork/lnd/pull/5047#discussion_r585751617.